src/core/i18n/translation/locales/en.ts

```diff
@@ -23,6 +25,7 @@ export const translation: ITranslations = {
                 send: 'Send',
                 tos: 'Terms of service',
                 transactions: 'Transactions',
+                to: 'to',
```

Comment:
change this to `To` and if you need them with `t` just use `.toLowerCase()`, this way we can reuse texts